libs/version.txt

```diff
@@ -8,6 +8,7 @@ dogpile.cache==1.1.5
 enzyme==0.4.1
 fese==0.1.2
 ffsubsync==0.4.20
+flask-cors >= 3.0.10
```

Comment:
Should be the actual version:
flask-cors==3.0.10

Reply:
Sharp! I've fixed it :)